pkg/controller/migplan/validation.go

```diff
@@ -1206,7 +1206,8 @@ func (r *NfsValidation) validate() error {
 	server := map[string]bool{}
 	for i := range r.Plan.Spec.PersistentVolumes.List {
 		pv := &r.Plan.Spec.PersistentVolumes.List[i]
-		if pv.NFS == nil {
+		// Skip non-NFS PVs and PVs which data should be copied
+		if pv.NFS == nil || pv.Selection.Action == "copy" {
```

Comment:
Fix looks sane, nice and simple. @aufi have you had a chance to test this yet?

Reply:
Sure, it worked. I dumped MigPlan content for both cases (with/without the change), https://gist.github.com/aufi/cb914d64a7aa0cbe8c3b57302dda7a2c. I didn't capture screencast video, but I could make it if needed. (only issue I had with testing was not reliable local execution of validations)

Reply:
YAML looks good. Adding my ACK 👍